F/FuzzifiED/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FuzzifiED"
+version = v"0.5.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mankai-chow/FuzzifiED_Fortran.git", "c96719aa5fb0e072612de78a1ebe2cf140515d47")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd FuzzifiED_Fortran/
+export MKL_THREADING_LAYER=GNU
```

Comment:
```suggestion
```
I don't see a dependency on MKL, and also it doesn't look like it is used, so lets remove it.